client/payment-details/summary/missing-order-notice/index.tsx

```diff
@@ -0,0 +1,106 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { Button, CardFooter } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import Loadable from 'wcpay/components/loadable';
+import { useState } from '@wordpress/element';
+import ConfirmationModal from 'wcpay/components/confirmation-modal';
+import CardNotice from 'wcpay/components/card-notice';
+
+/**
+ * Internal dependencies.
+ */
+
+declare const window: any;
+
+interface MissingOrderNoticeProps {
+	isLoading: boolean;
+	amount: string;
+}
+
+const MissingOrderNotice: React.FC< MissingOrderNoticeProps > = ( {
+	isLoading,
+	amount,
+} ) => {
+	const [ isModalOpen, setIsModalOpen ] = useState( false );
+
+	const handleOnButtonClick = () => {
+		setIsModalOpen( true );
+	};
+
+	const handleModalCancel = () => {
+		setIsModalOpen( false );
+	};
+
+	const handleModalConfirmation = () => {
+		// TODO: Handle the refund.
```

Comment:
Do you know if we plan to add Tracks events?

Reply:
I don't. And for the scope of PR I think we'll stop with empty handler until we have clarity on how the backed will work. I'll put this under some flag before merging.